tests/testthat/_snaps/fledgling.md

```diff
@@ -26,7 +26,7 @@
           },
           "section_state": "keep",
           "title": "fledge v1.0.0",
-          "version": "1.0.0",
+          "version": "v1.0.0",
           "date": "NA",
```

Comment:
Just leaving those out, or setting them to an explicit `NULL`, would be more idiomatic? What are the upsides of storing `NA` here?

Reply:
Currently, if we make them null, it makes this line fail https://github.com/cynkra/fledge/blob/40e192af2090f613370b04817bb7b25c16e66f7e/R/utils-parse-version.R#L3